pagesections/main.statement.section.js

```diff
@@ -136,6 +148,22 @@ const MainStatementSection = ( Base ) => class extends Base {
 		element.$( this.constructor.TOOLBAR_WIDGET_SELECTORS.SAVE_BUTTON ).click();
 	}
 
+	/**
+	 * Click cancel button in a statement element
+	 *
+	 * @private
+	 * @param {element} element
+	 */
+	clickCancelOnStatementElement( element ) {
+		var self = this;
+		element.$( this.constructor.TOOLBAR_WIDGET_SELECTORS.CANCEL_BUTTON ).waitUntil( function () {
+			return self.mainStatementsContainer.$(
```

Comment:
How does the element selected here differ from the element passed in? Shouldn't it be the same?

Reply:
Right, they are the same. I made a new change